apstra/api_design_templates_test.go

```diff
@@ -1094,7 +1094,7 @@ func TestRackBasedTemplateMethods(t *testing.T) {
 			return fmt.Errorf("asn allocation policy spine asn scheme mismatch expected %q got %q", req.AsnAllocationPolicy.SpineAsnScheme, rbt.AsnAllocationPolicy.SpineAsnScheme)
 		}
 
-		if !rackBasedTemplateFabricAddressingPolicyForbidden().Includes(apiVersion) {
+		if !rackBasedTemplateFabricAddressingPolicyForbidden().Includes(apiVersionString) {
```

Comment:
I wonder if it might make for smoother reading to have this accept the apiversion and for the Includes function to convert to string.

Reply:
The point of the change is to make it possible to eliminate this type of check.

In the future we'll use something like:

```go
if !rackBasedTemplateFabricAddressingPolicyForbidden.Check(client.apiVersion)
```

So, it'll come in a follow-up PR.